interfaces/Algorithmic.py

```diff
@@ -0,0 +1,166 @@
+import re
+import os
+import nltk
+import sqlite3
+import pickle
+
+from nltk.corpus import words, wordnet
+from autocorrect import Speller
+from pygments.token import Token
+from pygments.lexers import get_lexer_for_filename
+from sklearn.metrics.pairwise import cosine_similarity
+from sklearn.feature_extraction.text import TfidfVectorizer
+from interfaces.SemanticTest import SemanticTest
+from progress.bar import IncrementalBar
+
+class Algorithmic(SemanticTest):
+    def __init__(self):
+        try:
+            nltk.data.find('corpora/wordnet.zip')
+            nltk.data.find('corpora/words.zip')
+        except LookupError:
+            nltk.download('wordnet')
+            nltk.download('words')
+        self.conn = sqlite3.connect('transformed_texts.db')
+        self.c = self.conn.cursor()
```

Comment:
Why the database requests aren't handled by the SQLite class ?

Reply:
It would imply that i need to remove some abstractmethod from dbinterface if not all, so keeping it in a differnet file is doable but not much i think

Reply:
i think creating a new interface for that is better

Reply:
Yes great idea

Reply:
I don't see the new interface, have you created it ?